src/selinux_overview.md

```diff
@@ -68,17 +68,17 @@ and limited to the mimimum privileges required to do their job, so in a
     sandboxing services.
 9.  SELinux will not stop memory leaks or buffer over-runs (because its
     not designed to do this), however it may contain the damage that may
-    be done.
-10. SELinux will not stop all viruses/malware getting into the system
-    (as there are many ways they could be introduced (including by
+    be caused by this flaws.
```

Comment:
Should this be "... caused by these flaws."?

Reply:
Yes, it should be in the plural.